lib/domain.js

```diff
@@ -33,6 +33,10 @@ module.exports = function(logger) {
 
   return function domainMiddleware(req, res, next) {
     var requestDomain = domain.create();
+    //we need to explicitly bind req and res to the newly created domain
```

Comment:
Format comment to add space after // and to start with upper case:
// We ...
// See ...